File: CS/WinWebSolution.Module/DatabaseUpdater.cs
using System;
using DevExpress.ExpressApp;
using DevExpress.ExpressApp.Updating;

namespace WinWebSolution.Module {
    public class Updater : ModuleUpdater {
        public Updater(IObjectSpace objectSpace, Version currentDBVersion) : base(objectSpace, currentDBVersion) { }
        public override void UpdateDatabaseAfterUpdateSchema() {
            base.UpdateDatabaseAfterUpdateSchema();
            TestObject obj1 = ObjectSpace.CreateObject<TestObject>();
            obj1.NonRestrictedProperty = "NonRestrictedProperty";
            obj1.CannotSortAndGroupByThisProperty = "CannotSortAndGroupProperty";
            obj1.Save();
            TestObject obj2 = ObjectSpace.CreateObject<TestObject>();
            obj2.NonRestrictedProperty = obj1.NonRestrictedProperty;
            obj2.CannotSortAndGroupByThisProperty = obj1.CannotSortAndGroupByThisProperty;
            obj2.Save();
        }
    }
}

File: VB/WinWebSolution.Module/DatabaseUpdater.vb
Imports Microsoft.VisualBasic
Imports System
Imports DevExpress.ExpressApp
Imports DevExpress.ExpressApp.Updating

Namespace WinWebSolution.Module
	Public Class Updater
		Inherits ModuleUpdater
		Public Sub New(ByVal objectSpace As IObjectSpace, ByVal currentDBVersion As Version)
			MyBase.New(objectSpace, currentDBVersion)
		End Sub
		Public Overrides Sub UpdateDatabaseAfterUpdateSchema()
			MyBase.UpdateDatabaseAfterUpdateSchema()
			Dim obj1 As TestObject = ObjectSpace.CreateObject(Of TestObject)()
			obj1.NonRestrictedProperty = "NonRestrictedProperty"
			obj1.CannotSortAndGroupByThisProperty = "CannotSortAndGroupProperty"
			obj1.Save()
			Dim obj2 As TestObject = ObjectSpace.CreateObject(Of TestObject)()
			obj2.NonRestrictedProperty = obj1.NonRestrictedProperty
			obj2.CannotSortAndGroupByThisProperty = obj1.CannotSortAndGroupByThisProperty
			obj2.Save()
		End Sub
	End Class
End Namespace
